frontend/tailwind.config.js

```diff
@@ -5,7 +5,27 @@ module.exports = {
     './components/**/*.{js,ts,jsx,tsx}',
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'primary': {
+          DEFAULT: '#a3dffd ',
+          dark:'#333049'
```

Comment:
As discussed before, the dark theme colors do not conform to the current OSF color palette. I suggest changing this to the OSF colors with the accent color tweaked for accessibility/contrast

Reply:
Rejected